package/MDAnalysis/analysis/helix_analysis.py

```diff
@@ -33,23 +33,19 @@
 
 This module contains code to analyse protein helices using the
 HELANAL_ algorithm
-([Bansal2000]_ , [Sugeta1967]_ ).
+:cite:p:`Bansal2000` ([Sugeta1967]_ ).
```

Comment:
Can you turn Sugeta1967 also into a bib link? It seems inconsistent to mix the two.

Reply:
Oh, thanks for catching this, completely missed it.